release.py

```diff
@@ -101,7 +107,16 @@
     # Assuming twine is installed.
     print("### twine check")
     subprocess.run(["twine", "check"] + DISTFILES, check=True)
+except subprocess.CalledProcessError as e:
+    print("ERROR: Last step returned exitcode != 0")
+    sys.exit(e.returncode)
+
+choice = input("Ready to upload to PyPI? [y/N]: ")
+if choice.casefold() not in ("y", "yes"):
+    print("Okay.")
+    sys.exit(0)
```

Comment:
Final chance to change our mind. For example, if we use `release.py` just for building and not yet ready to release.